docs/Reference/API-Methods.md

```diff
@@ -0,0 +1,2912 @@
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+          "jsonrpc":"2.0",
+          "id":67,
+          "result":"0x0000000000000000000000000000000000000000000000000000000000000000"
+        }
+        ```
+
+    If you provide the callback URL, you receive the following response after submitting the transaction.
+    This example assumes a webserver that can be accessed by calling http://localhost:8080 has been set up to accept
+    POST requests:
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"eth_sendTransactionAsync","params":[{"from":"0xed9d02e382b34818e88b88a309c7fe71e65f419d","data":"0x6060604052341561000f57600080fd5b604051602080610149833981016040528080519060200190919050505b806000819055505b505b610104806100456000396000f30060606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680632a1afcd914605157806360fe47b11460775780636d4ce63c146097575b600080fd5b3415605b57600080fd5b606160bd565b6040518082815260200191505060405180910390f35b3415608157600080fd5b6095600480803590602001909190505060c3565b005b341560a157600080fd5b60a760ce565b6040518082815260200191505060405180910390f35b60005481565b806000819055505b50565b6000805490505b905600a165627a7a72305820d5851baab720bba574474de3d09dbeaabc674a15f4dd93b974908476542c23f00029000000000000000000000000000000000000000000000000000000000000002a","gas":"0x47b760","privateFor":["ROAZBWtSacxXQrOe3FGAqJDyJjFePR5ce4TSIzmJ0Bc="],"callbackUrl":"http://localhost:8080"}],"id":67}'
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+          "id":67,
+          "txHash":"0x75ebbf4fbe29355fc8a4b8d1e14ecddf0228b64ef41e6d2fce56047650e2bf17"
+        }
+        ```
+
+### `getContractPrivacyMetadata`
+
+Queries the privacy metadata for the specified contract account address.
+
+#### Parameter
+
+*string* - contract address
+
+#### Returns
+
+* `creationTxHash`: *data* - affected contract's original transaction's encrypted payload hash
+
+* `privacyFlag`: *number* - unsigned integer with 1 for counterparty protection and 3 for private state validation contracts
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"eth_getContractPrivacyMetadata","params":["0x1932c48b2bf8102ba33b4a6b545c32236e342f34"],"id":15}' --header "Content-Type: application/json"
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+          "jsonrpc":"2.0",
+          "id":15,
+          "result": {
+            "creationTxHash":[246,124,116,139,190,217,33,16,203,102,81,13,65,58,249,145,68,180,67,79,163,37,119,27,99,35,247,240,12,53,25,45,47,134,16,118,246,128,97,237,45,50,79,97,78,221,47,1 89,11,165,238,36,8,187,66,64,42,135,108,75,41,85,152,183],
```

Comment:
is it really a data passed as a bytes array here? It would seem more consistant that it's an hex string here.
This is probably the geth console return value but to confirm this looks the same in the JSON-RPC return.
See for instance how data is described in eth_sendTransactionAsync